navigation.yaml

```diff
@@ -39,6 +39,15 @@ openstack:
   children:
     - title: Overview
       path: /openstack
+    - title: Install
+      path: /openstack/install
+
+      children:
```

Comment:
these are backwards... and not appearing

Reply:
I've fixed the backwardness, but not them not appearing. This is a bug with the current state of the secondary nav (basically I forgot tertiary breadcrumbs are a thing) but they'll be fixed in #2771